website/docs/specification/Architecture.md

```diff
@@ -13,12 +13,85 @@ The architecture is the blueprint for all the components of the specification an
 
 ## Principles
 
+The architecture principles that guide the UNTP design are 
 
-## Each Actor
+|Name|Principle|Rationale|
+|--|--|--|
+|No dependency |UNTP should not require any collaboration or dependency between the issuer of DPPs and the consumer / verifier|Imposing such collaboration as a pre-requisite for action in a complex many-to-many ecosystem would essentially stall progress|
+|Unknown verifier |UNTP should not assume that that the consumer / verifier of UNTP data is known to the issuer, even when confidential data access is required|In a decentralised architecture with thousands of issuers, it would be impractical to register every authorised verifier with every issuer.|
+|Any maturity|UNTP should not assume any technical maturity for verifiers|DPPs and other credentials must work equally for human and machine verifiers - otherwise an insurmountable complexity of knowing which customer has what capability would be required|
+|Legacy data carriers |UNTP should work with any carrier of a product identifier including iD barcodes, RFID, 2D codes, digital documents|1D barcodes and RFIDs are ubiquitous and will only be replaced slowly.  Uptake should not require manufacturers to re-instrument their production lines and printing processes|
```

Comment:
This new work item proposition at the w3c caught my eyes and is directly related to this:
https://digitalbazaar.github.io/vc-barcodes/